package.json

```
@@ -3559,23 +3559,23 @@
                 "command": "aws.codeWhisperer",
                 "key": "alt+c",
                 "mac": "alt+c",
-                "when": "editorTextFocus && CODEWHISPERER_ENABLED"
+                "when": "editorTextFocus && (CODEWHISPERER_ENABLED || isCloud9)"
             },
             {
                 "command": "aws.codeWhisperer.rejectCodeSuggestion",
                 "key": "escape",
                 "mac": "escape",
-                "when": "inlineSuggestionVisible && !editorReadonly && CODEWHISPERER_ENABLED"
+                "when": "inlineSuggestionVisible && !editorReadonly && (CODEWHISPERER_ENABLED || isCloud9)"
```

and because parens aren't supported, it's clearer to put the `isCloud9` at the start:
```suggestion
                "when": "isCloud9 || inlineSuggestionVisible && !editorReadonly && CODEWHISPERER_ENABLED"
```